cardinal/constants/game_constants.go

```diff
@@ -0,0 +1,48 @@
+package constants
+
+// Define the constants for GameConstants
+const (
+	// MAX_SIZES for functions
+	MAX_TOK uint8 = 16
+	MIN_TOK uint8 = 2
+
+	// DATA_BITS for packing
+	TERRAIN_BITS uint8 = 24 // << 24
+	ROOM_BITS    uint8 = 16 // << 16
+	OBJECT_BITS  uint8 = 8  // << 8
+
+	// Direction bits
+	NORTH_DIR uint8 = 1 // 0x0001
+	EAST_DIR  uint8 = 2 // 0x0010
+	SOUTH_DIR uint8 = 4 // 0x0100
+	WEST_DIR  uint8 = 8 // 0x1000
+
+	SIZED_AR_SIZE uint32 = 32 // the max size of a Sized Array, 31 items + 1 for count
+)
+
+// Define the constants for ErrCodes
+const (
```

Comment:
these should probably in fact be errors see https://go.dev/blog/error-handling-and-go we use these constatnt because as usual Solidity. Try to be idiomatic in Go this is a Go project. The LOGIC of the Solidity version of the engine is important not the code. You can use the codes in an actual error so we will leave these in. I would give them more meaningful names. Either that or we need a handler that generates a decent message based on the code.

Reply:
Done, create a set of errors that have a struct of code and message that will make easy to access them. I changed a the err variable on the verbData struct to match the errors